TMessagesProj/config/debug/AndroidManifest_SDK23.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
you can leave it there, it's just not needed and doing nothing with the removal of all those receivers, services and permissions